R/course.R

```diff
@@ -263,11 +263,12 @@ normalize_url <- function(url) {
 }
 
 conspicuous_place <- function() {
-  Filter(dir.exists, c(
-    file.path(Sys.getenv("HOME"), "Desktop"), # typical macOS = ~/Desktop
-    file.path(Sys.getenv("USERPROFILE"), "Desktop"), # typical Windows Desktop
-    "~",
-    getwd()
+  Filter(dir_exists, c(
+    path(Sys.getenv("HOME"), "Desktop"), # typical macOS = ~/Desktop
```

Comment:
This is probably redundant with next line.

Reply:
Yeah, I think you should remove that line now (and the comment on the next line)